robottelo/config/__init__.py

```diff
@@ -133,8 +133,8 @@ def configure_nailgun():
 
     def patched_gpgkey_init(self, server_config=None, **kwargs):
         """Set a default value on the ``content`` field."""
-        gpgkey_init(settings, server_config, **kwargs)
-        settings._fields['content'].default = str(
+        gpgkey_init(self, server_config, **kwargs)
+        self._fields['content'].default = str(
```

Comment:
Shouldn't this be setting `_fields` on the GPGKey entity instance returned by calling `entities.GPGKey.__init__`? The `gpgkey_init()` return needs to be captured and referenced here?

Reply:
Taking a second look, `self` is just _VERY_ misleading in this context.

Reply:
discussed in chat

Reply:
The ugliness of this baby won't be fully addressed here.